server/plugin/client_test.go

```diff
@@ -0,0 +1,294 @@
+package plugin
+
+import (
+	"errors"
+	"io"
+	"net/http"
+	"net/url"
+	"reflect"
+	"testing"
+
+	"bou.ke/monkey"
+	"github.com/Brightscout/mattermost-plugin-azure-devops/server/serializers"
+	"github.com/mattermost/mattermost-server/v5/plugin/plugintest"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClientGenerateOAuthToken(t *testing.T) {
+	defer monkey.UnpatchAll()
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	p.API = mockAPI
+	c := InitClient(&p)
+	p.Client = c
+	for _, testCase := range []struct {
+		description string
+		err         error
+		statusCode  int
+	}{
+		{
+			description: "test ClientGenerateOAuthToken",
```

Comment:
keep the test description consistent 
follow the rule mentioned here
https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/47#pullrequestreview-1096035931